docs/application/vstools/install.md

```diff
@@ -16,7 +16,7 @@ You need the following components on top of Visual Studio to make Visual Studio
 To work with Visual Studio Tools for Tizen, your computer must have:
 
 - At least 1.5 GB of available disk space
-- Visual Studio 2017
+- Visual Studio 2017 for Tizen-4.0 and Tizen-5.0 or Visual Studio 2019 for Tizen-4.0 and higher
```

Comment:
Suggestion:
- Visual Studio 2017 to use Tizen 4.0 and 5.0
- Visual Studio 2019 to use Tizen 4.0 and higher